src/hooks/useOldAssetsDetected.ts

```diff
@@ -7,7 +7,12 @@ export const useOldAssetsDetected = () => {
   const { activeChain = { id: 1 } } = useNetwork();
 
   return useAppSelector(state => {
-    if (activeChain.id && (activeChain.id === NetworkId.MAINNET || activeChain.id === NetworkId.TESTNET_RINKEBY)) {
+    if (
+      activeChain.id &&
+      (activeChain.id === NetworkId.MAINNET ||
```

Comment:
You've tested for these three a few times. I would suggest extracting a function to test if the chain is ethereum (with a flag for allowing testnets?).
e.g.
`export function isChainEthereum(chainId: number, includeTestnets: boolean): boolean`

Reply:
✅